tests/framework/installer/ceph_manifests.go

```diff
@@ -1938,10 +1938,18 @@ spec:
               properties:
                 bucket:
                   properties:
-                    enabled:
+                    disabled:
                       type: boolean
                     interval:
                       type: string
+                    timeout:
+                      type: string
+                  livenessProbe:
```

Comment:
The indentation on this looks at the same level as the `bucket.properties`. It should instead be at the same level as `bucket`, right?